packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -126,7 +126,22 @@ export const messages = {
   force_stop_apex_debug_logging:
     'SFDX: Turn Off Apex Debug Log for Replay Debugger',
   isv_debug_bootstrap_step1_create_project:
-    'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
-  isv_debug_bootstrap_step1_configure_project:
-    'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project'
+    'SFDX: ISV Debugger Setup, Step 1 of 7: Create Project',
```

Comment:
cc @ruthemmanuelle to review these messages.

Reply:
@guw - When I reviewed the messages for steps 1 and 2, I thought they'd be available commands in VS Code. I subsequently saw that they showed in the output but not in the command palette. Do all seven of these messages show only in the output? If so, here are the messages I suggest:
'SFDX: ISV Debugger Setup, Step 1 of 7: Creating project',
'SFDX: ISV Debugger Setup, Step 2 of 7: Configuring project',
'SFDX: ISV Debugger Setup, Step 3 of 7: Retrieving unpackaged Apex code',
'SFDX: ISV Debugger Setup, Step 4 of 7: Converting unpackaged Apex code',
'SFDX: ISV Debugger Setup, Step 5 of 7: Querying for installed packages',
'SFDX: ISV Debugger Setup, Step 6 of 7: Retrieving packages',
'SFDX: ISV Debugger Setup, Step 7 of 7: Converting package: %s',

Let me know what you think of these.

Reply:
@ruthemmanuelle They show up in the output *and* in the Running Tasks view in VS Code.